_episodes/03-high-dim-modeling.md

```diff
@@ -4,43 +4,121 @@ teaching: 0
+      * Interpretability: How to determine which features are most relevant in your model?
+          * Linear models: coefficient size 
+      * Compare performance across all feature selection methods using a new dataset. Questions to consider....
+          * Which feature selection method yielded the best performing model?
+          * When might you choose one feature selection over another?
+          *  
+
+# Possible Datasets (internal use; will remove this later)
+1. Ames house prices dataset (sklearn: https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_boston.html)
+   * 1460 observations, 80 feautres (bingo!)
+2. Boston house prices dataset (sklearn)
+   * 506 observations, 14 different features; easy to fit regression model to
+   * Evidently there's an ethical issue with this dataset. Recommended to not use (https://scikit-learn.org/stable/modules/generated/sklearn.datasets.load_boston.html).
+3. Diabetes
+   * 442 observations, 10 features, easy to fit regression model to
+
+# Functions Provided
+1. Load/plot data
 
 # Introduction
-- Curse of dimensionality
-- Bias VS Variance, 10X as many datapoints as features as general rule of thumb
-- Methods for feature selection
+## Machine learning — a brief overview
+Machine learning is the process of learning (optimizing) a function that is able to map some input to a desired output.
 
-> ## Exercise - demonstrate overfitting (might just show this in intro, but leaving here as placeholder for now)
-> 
-> 
->
-> > ## Solution
-> >  
-> >  
-> {: .solution}
-{: .challenge}
+**Find/make diagram**: input data --> cool algorithms --> prediction forecast, recommendation ,decision
 
-> ## Exercise
-> In this example, we would like to classify images of cats versus dogs. In every image example, a cat or a dog appears at the center of the image with some background imagery present as well. There are two example images provided below. Instead of training our model on every pixel present in each image, what could we do to help the model hone in on the important aspects of the images that relate to how dogs and cats differ?
-> 
->
-> > ## Solution
-> >  - Include only the center of each image--where a dog or a cat appears
-> >  - Include only pixels that contain the head of the animal--where differences are more noticeable between the species.
-> {: .solution}
-{: .challenge}
+The techniques breakdown into two broad categories, predictors and classifiers. Predictors are used to predict a value (or set of value) given a set of inputs, for example trying to predict the cost of something given the economic conditions and the cost of raw materials or predicting a country’s GDP given its life expectancy. Classifiers try to classify data into different categories, for example deciding what characters are visible in a picture of some writing or if a message is spam or not. Some examples might include:
+* predicting a person’s weight based on their height
```

Comment:
Don't know if this one is a good one to include as weight is a feature individuals experience discrimination for.